org.code.toboggan.ui/src/org/code/toboggan/ui/dialogs/extensions/DialogWSEvent.java

```diff
@@ -14,8 +20,25 @@
 import org.eclipse.swt.widgets.Display;
 import org.eclipse.swt.widgets.Shell;
 
+import clientcore.dataMgmt.SessionStorage;
+
 public class DialogWSEvent implements IWSEvent {
 	private Logger logger = LogManager.getLogger(this.getClass());
+	
+	public DialogWSEvent(){
+		CoreActivator.getSessionStorage().addPropertyChangeListener(new PropertyChangeListener() {
+			
+			@Override
+			public void propertyChange(PropertyChangeEvent evt) {
+				if (evt.getPropertyName().equals(SessionStorage.USERNAME)){
+					Display.getDefault().asyncExec(() -> {
+						Shell shell = Display.getDefault().getActiveShell();
+						new ConfirmSubscribeDialog(shell).open();
+					});
+				}				
+			}
+		});
```

Comment:
Hm, I don't know if this is really the right place for this code to go since it's not really a websocket event. My suggestion is to either hook this up via an extension point in the Core plugin (a la ON_CONNECT in the Network plugin triggering the onConnect method below) or for a quicker fix just throw it in the UI activator in a "register listeners" sort of method.